modules/statusUpdate.js

```diff
@@ -0,0 +1,23 @@
+import Store from "./store";
+
+export default class StatusUpdate {
+    static updateAllCheckbox = () => {
+        return document.querySelectorAll('.task-complete');
+    }
+
+    static completedCheckbox = () => {
+        const updateComplete = document.querySelectorAll('.task-complete');
+        const tasksList = Store.getLocalStorage();
+        updateComplete.forEach((box, index) => {
+        
+        if(box.checked){
+          tasksList[index].completed = true;
+          localStorage.setItem('tasks', JSON.stringify(tasksList));
+        } else {
+          tasksList[index].completed = false;
+          localStorage.setItem('tasks', JSON.stringify(tasksList));
+        }
```

Comment:
- [OPTIONAL] I noticed that after checking a task as completed, it indeed changed from false to true on the browser and in the local storage thou, but the task becomes unchecked on the browser after page reload. Also, the true value in the local storage turns false from true if another task is checked. This is part of requirements to get approval for this milestone. But I marked it optional because it was not mentioned by the previous reviewer. It is therefore highly recommended to look into this issue and fix it.
- Please, take a look :point_down: 
[completed-issue.webm](https://user-images.githubusercontent.com/26597432/213566172-35627811-ac17-45a1-897e-267c86635a1d.webm)

Reply:
Okay. Thank you. I will look into it